src/txgraph.cpp

```diff
@@ -1854,6 +1865,33 @@ TxGraph::GraphIndex TxGraphImpl::CountDistinctClusters(std::span<const Ref* cons
     return ret;
 }
 
+std::pair<std::vector<FeeFrac>, std::vector<FeeFrac>> TxGraphImpl::GetMainStagingDiagrams() noexcept
+{
+    Assume(m_clustersets.size() >= 2);
```

Comment:
Shouldn't this be <= instead.
```suggestion
    Assume(m_clustersets.size() <= 2);

```